README.md

```diff
@@ -4,26 +4,26 @@
 ALF(Alice Low-level Frontend) spawns DIM services as an interface with detector FEEs(Front-End Electronics). The DIM services can be accessed through DCS's [FRED](https://gitlab.cern.ch/alialfred/FREDServer).
 
 ## Requirements
-In order to run ALF a DIM Nameserver has to be up and running.
+In order to run ALF a DIM Nameserver has to be up and running. For performance reasons, it is recommended to run the DIM Nameserver on the host running the FRED server.
 
 ## Usage
 
 ### o2-alf
 o2-alf is the binary of the ALF server. The only option it expects is the address of the DIM Nameserver. It can be passed either as a command-line argument or as an environmental variable.
 
 `
-o2-alf --dim-dns-node thedimdns
+o2-alf --dim-dns-node thedimdns.cern.ch
 `
 
 `
-DIM_DNS_NODE=thedimdns o2-alf
+DIM_DNS_NODE=thedimdns.cern.ch o2-alf
 `
 
 ### o2-alf-client
 o2-alf-client is the binary of an ALF client used solely for testing purposes. On top of the DIM Nameserver it expects the hostname of the node hosting the ALF server, the card's serial and the link number as command-line arguments.
 
 `
-o2-alf-client --dim-dns-node thedimdns --alf-id thealfserver --serial 0 --link 4
+o2-alf-client --dim-dns-node thedimdns.cern.ch --alf-id thealfserver.cern.ch --serial 0 --link 4
```

Comment:
The --alf-id option serves to help construct the service names. Since for that the hostname is not the FQDN it should remain as is.